tests/dbt/test_graph.py

```diff
@@ -429,7 +429,7 @@ def test_load_via_dbt_ls_does_not_create_target_logs_in_original_folder(
 
     used_cwd = Path(mock_popen.call_args[0][0][-5])
```

Comment:
@pankajastro could you confirm the value of `used_cwd`, please?

Reply:
When the cache profile is enabled 
local mac used_cwd: `PosixPath('/var/folders/t4/sk99_rj509127895hqk1m9x00000gn/T/cosmos/profile/470f1452ea88562203ec89a49bab7e2c6043ef3b7f93e61ce30772e24aa38289')`
CI used_cwd: `PosixPath('/tmp/cosmos/profile/470f1452ea88562203ec89a49bab7e2c6043ef3b7f93e61ce30772e24aa38289'`

Reply:
As we discussed, we've changed to `Path(mock_popen.call_args[0][0][5])`